src/pybind/mgr/cephadm/module.py

```diff
@@ -141,8 +141,7 @@ def rm(self, service_name):
             del self.spec_created[service_name]
             self.mgr.set_store(SPEC_STORE_PREFIX + service_name, None)
 
-    def find(self, service_name=None):
-        # type: (Optional[str]) -> List[ServiceSpec]
+    def find(self, service_name: Optional[str] = None) -> List[Any]:
```

Comment:
```suggestion
    def find(self, service_name: Optional[str] = None) -> List[ServiceSpec]:
```
got it:

```python
drive_groups: List[DriveGroupSpec] = self.spec_store.find(service_name=drive_group_name)
```
would fail otherwise. What about:
```python
drive_groups = typing.cast(List[DriveGroupSpec], self.spec_store.find(service_name=drive_group_name))
```
?